using-mobile-extensions/mobile-core/signals/signals-extension-and-rules-engine-integration.md

```diff
@@ -12,21 +12,21 @@ To send PII data to external destinations, the `PII` action can trigger the Rule
 
 ## Rules tokens <a id="rules-tokens"></a>
 
-Tokens are special strings used in rule actions as values, which are expanded by the SDK when the action is carried out. The format of a token is `{%%token%%}`, where token is any data element defined in Adobe Launch for a mobile property which identifies the source of the data from which the token is expanded. For example, `{%%My Data element for ECID%%}` can be used in the Signal Post back action, where `My Data element for ECID` is a Data element created using the Mobile Core extension, and Data element type Experience Cloud ID.
+Tokens are special strings that are used in rule actions as values and are expanded by the SDK when the action is carried out. The format of a token is `{%%token%%}`, where token is any data element that is defined in Adobe Experience Platform Launch for a mobile property that identifies the source of the data from which the token is expanded. For example, `{%%My Data element for ECID%%}` can be used in the Signal Postback action, where `My Data element for ECID` is a data element that was created using the Mobile Core extension, and Data element type is Experience Cloud ID.
```

Comment:
In the 3rd sentence:

- Please lower case "Postback"
- Please lower case "Data" in "Data element type"
- Please add "the" before "data"

Reply:
Implemented